openssl-dynamic/src/main/c/error.c

```diff
@@ -37,18 +37,13 @@
  */
 void tcn_ThrowException(JNIEnv *env, const char *msg)
 {
-    jclass javaExceptionClass;
-
-    javaExceptionClass = (*env)->FindClass(env, "java/lang/Exception");
-    if (javaExceptionClass == NULL) {
-        fprintf(stderr, "Cannot find java/lang/Exception class\n");
-        return;
-    }
-    (*env)->ThrowNew(env, javaExceptionClass, msg);
-    (*env)->DeleteLocalRef(env, javaExceptionClass);
-
+    (*env)->ThrowNew(env, tcn_get_exception_class(), msg);
```

Comment:
the pattern we have been using to partition things is to have the module export a method which is called on ini load and a method for jni unload (e.g. `netty_internal_tcnative_error_JNI_OnLoad` and `netty_internal_tcnative_error_JNI_OnUnLoad`). This will avoid having to expose accessor methods (e.g. `tcn_get_exception_class()` and `tcn_get_illegal_argument_exception_class()`) which are only intended to be used in the "error" module. can we use the same approach here too?